docs/resources/vault_secrets_integration_postgres.md

```diff
@@ -0,0 +1,57 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_integration_postgres Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets Postgres integration resource manages a Postgres integration.
+---
+
+# hcp_vault_secrets_integration_postgres (Resource)
+
+The Vault Secrets Postgres integration resource manages a Postgres integration.
+
+## Example Usage
+
+```terraform
+resource "hcp_vault_secrets_integration_postgres" "example" {
+  name         = "my-postgres-1"
+  capabilities = ["ROTATION"]
+  static_credential_details = {
+    connection_string = "postgres://user:password@localhost:5432/dbname"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `capabilities` (Set of String) Capabilities enabled for the integration. See the Vault Secrets documentation for the list of supported capabilities per provider.
+- `name` (String) The Vault Secrets integration name.
+
+### Optional
+
+- `project_id` (String) HCP project ID that owns the HCP Vault Secrets integration. Inferred from the provider configuration if omitted.
+- `static_credential_details` (Attributes) Postgres connection string (DSN) to the target database. (see [below for nested schema](#nestedatt--static_credential_details))
+
+### Read-Only
+
+- `organization_id` (String) HCP organization ID that owns the HCP Vault Secrets integration.
+- `resource_id` (String) Resource ID used to uniquely identify the integration instance on the HCP platform.
+- `resource_name` (String) Resource name used to uniquely identify the integration instance on the HCP platform.
+
+<a id="nestedatt--static_credential_details"></a>
+### Nested Schema for `static_credential_details`
+
+Required:
+
+- `connection_string` (String, Sensitive) Connection string (DSN) for the Postgres database.
+
+## Import
+
+Import is supported using the following syntax:
+
+```shell
+terraform import hcp_vault_secrets_integration_postgres.example my-postgres-1
```

Comment:
Please clarify ` my-postgres-1`

Reply:
for instance: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc#import